helper-scripts/README.md

```diff
@@ -0,0 +1,13 @@
+# Helper scripts
+
+## delete_stack.sh
+
+This script provides the ability to:
+- empty and delete any versioned S3 buckets (if present in a SAM application)
+- delete the SAM application
+
+```bash
+export STACK_NAME=backend-stack-name    (REQUIRED)
+export TMP_DIR=./tmp                    (OPTIONAL)
```

Comment:
If optional, why and when to use it?

Reply:
it would be to update where you would like the aws cli output to go. I've removed in any case